SPECS/librsync/librsync.spec

```diff
@@ -58,6 +59,9 @@ make %{?_smp_mflags}
 %install
 cd build
 make DESTDIR=%{buildroot} install
+# install outside of DESTDIR to resolve test error:
+# /usr/src/mariner/BUILD/librsync-2.0.2/build/rdiff: error while loading shared libraries: librsync.so.2: cannot open shared object file: No such file or directory
+make install
```

Comment:
We could also surround this step with `%if %{with_check}` to make the build faster when we're not running tests.

Reply:
sure, done